src/Madara.ts

```diff
@@ -356,7 +370,7 @@ export abstract class Madara extends Source {
     /**
      * Constructs requests to be sent to the Madara /admin-ajax.php endpoint.
      */
-    constructAjaxHomepageRequest(page: number, postsPerPage: number, meta_key: string): any {
+    constructAjaxHomepageRequest(page: number, postsPerPage: number, meta_key: string, meta_value = ''): any {
```

Comment:
Why an empty string? Wouldn't it be better to use the optional argument, and pass `undefined` on to the request handler? I think this is better:
```suggestion
    constructAjaxHomepageRequest(page: number, postsPerPage: number, meta_key: string, meta_value?: string): any {
```